src/components/tooltips/branch/twt-tap-changer-characteristics.tsx

```diff
@@ -35,7 +35,7 @@ export const TwtTapChangerCharacteristics: React.FC<TwtTapChangerCharacteristics
         rows.push({
             tap: phaseTapChanger.tapPosition ?? '-',
             rho: '-',
-            alpha: phaseTapChanger.steps[phaseTapChanger.tapPosition].alpha,
+            alpha: phaseTapChanger.steps[phaseTapChanger.tapPosition - 1].alpha,
```

Comment:
```suggestion
            alpha: phaseTapChanger.steps[phaseTapChanger.tapPosition - phaseTapChanger.lowTapPosition].alpha,
```
otherwise if it starts at 2 it does not work either...